cmd/kubeadm/app/phases/kubelet/flags_test.go

```diff
@@ -189,3 +190,81 @@ func TestGetNodeNameAndHostname(t *testing.T) {
 		})
 	}
 }
+
+func TestReadKubeadmFlags(t *testing.T) {
+	tests := []struct {
+		name          string
+		fileContent   string
+		expectedValue []string
+		expectError   bool
+	}{
+		{
+			name:          "valid kubeadm flags with container-runtime-endpoint",
+			fileContent:   `KUBELET_KUBEADM_ARGS="--container-runtime-endpoint=unix:///var/run/containerd/containerd.sock --pod-infra-container-image=registry.k8s.io/pause:3.10"`,
```

Comment:
instead of `registry.k8s.io/pause:3.10` use something like `registry.k8s.io/pause:1.0`. that is to avoid future PRs trying to update this version after we update the k8s pause image. change this in all tests please.